library.properties

```diff
@@ -1,5 +1,5 @@
 name=Amperka Octoliner
-version=2.0.0
+version=2.0.1
```

Comment:
It adds a feature. So, should be `2.1.0`, not `2.0.1`. Patch bumps are only for bug fixes

Reply:
:exclamation:

Reply:
done